### ISSUE TITLE
Decoding observations accumulating 

### ISSUE THREAD

**Issue by ArgoNavi** [dernasherbrezon/r2cloud#136]
Over the past few days I've been having a problem with observations taking hours (6+) to decode, the result is an unending accumulation of observations that are decoding.

Currently the only way to fix this is to disable observations and wait several hours (8+) for everything to finish, however once enabled again the accumulation begins.

METEOR-M 2 is disabled.

<img width="474" alt="Screenshot 2022-12-11 at 10 53 08 PM" src="https://user-images.githubusercontent.com/63644530/206957011-32d080d4-a064-48f0-a034-d334e76427f9.png">

**Comment by ArgoNavi** [comment]
Here are the Logs


```shell
-- Logs begin at Thu 2019-02-14 05:11:59 EST, end at Sun 2022-12-11 18:44:08 EST. --
Dec 11 17:08:00 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/39430/data/1670777717179-39430-0/data.bin [ru.r2cloud.satellite.ObservationDao saveData]
Dec 11 17:08:00 r2cloud java[18571]: [1670777717179-39430-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:08:01 r2cloud java[18571]: stopping: rtl_sdr for 1670795909835-46912-0 [ru.r2cloud.util.Util shutdown]
Dec 11 17:08:01 r2cloud java[18571]: Signal caught, exiting!
Dec 11 17:08:01 r2cloud java[18571]: [1670777717179-39430-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:08:01 r2cloud java[18571]: [1670774247643-51044-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:08:01 r2cloud java[18571]: [1670795909835-46912-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:08:02 r2cloud java[18571]: User cancel, exiting...
Dec 11 17:08:02 r2cloud java[18571]: [1670795909835-46912-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 17:08:02 r2cloud java[18571]: deleting old observation: ./data/satellites/52191/data/1670518345760-52191-0 last update time: Thu Dec 08 17:01:15 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 17:08:02 r2cloud java[18571]: deleting old observation: ./data/satellites/43132/data/1670518981461-43132-0 last update time: Thu Dec 08 17:10:31 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 17:17:45 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 437462989 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/40014-1670797065797-40014-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 17:17:45 r2cloud java[18571]: [1670797065797-40014-0] moving rotator to Position [azimuth=123.31903499881038, elevation=5.0000369676605105] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:17:45 r2cloud java[18571]: Found 1 device(s):
Dec 11 17:17:45 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 17:17:45 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 17:17:46 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 17:17:46 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 17:17:46 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 17:17:46 r2cloud java[18571]: Tuned to 437462989 Hz.
Dec 11 17:17:46 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 17:17:46 r2cloud java[18571]: Reading samples in async mode...
Dec 11 17:23:13 r2cloud java[18571]: loading satellites from leosatdata [ru.r2cloud.cloud.LeoSatDataClient loadSatellites]
Dec 11 17:23:14 r2cloud java[18571]: no satellite updates from leosatdata [ru.r2cloud.cloud.LeoSatDataClient loadSatellites]
Dec 11 17:23:14 r2cloud java[18571]: loading tle from: http://mstl.atl.calpoly.edu/~ops/keps/kepler.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:15 r2cloud java[18571]: received tle for 136 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:15 r2cloud java[18571]: loading tle from: http://celestrak.com/NORAD/elements/satnogs.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:15 r2cloud java[18571]: received tle for 252 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:15 r2cloud java[18571]: loading tle from: http://celestrak.com/NORAD/elements/active.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:16 r2cloud java[18571]: received tle for 6847 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 17:23:16 r2cloud java[18571]: check for not processed observations [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670784052694-40042-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670792393555-39444-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670784627898-46914-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670789782198-46912-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670780702937-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670794241800-43197-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670786458317-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670792247245-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670795909835-46912-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670791605184-44419-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670779377193-43743-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670785351241-49017-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670793766986-43196-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670786919587-46493-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670794515469-52900-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670778970469-43721-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:23:16 r2cloud java[18571]: resuming decoding: 1670790683316-46914-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 17:25:35 r2cloud java[18571]: [1670774247643-51044-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:25:35 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/51044/data/1670774247643-51044-0/data.bin [ru.r2cloud.satellite.ObservationDao saveData]
Dec 11 17:25:35 r2cloud java[18571]: [1670774247643-51044-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:25:37 r2cloud java[18571]: [1670774247643-51044-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:25:37 r2cloud java[18571]: [1670778238517-28654-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:25:37 r2cloud java[18571]: started with arguments: [wxtoimg, -e, HVC, -t, n, -c, -o, /home/pi/r2cloud/./data/satellites/28654/data/1670778238517-28654-0/output.wav, /tmp/apt-1670778238517-28654-
Dec 11 17:25:37 r2cloud java[18571]: Satellite: NOAA [ru.r2cloud.satellite.decoder.APTDecoder$1 run]
Dec 11 17:25:49 r2cloud java[18571]: stopping: rtl_sdr for 1670797065797-40014-0 [ru.r2cloud.util.Util shutdown]
Dec 11 17:25:49 r2cloud java[18571]: Signal caught, exiting!
Dec 11 17:25:49 r2cloud java[18571]: [1670797065797-40014-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:25:50 r2cloud java[18571]: User cancel, exiting...
Dec 11 17:25:50 r2cloud java[18571]: [1670797065797-40014-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 17:25:50 r2cloud java[18571]: deleting old observation: ./data/satellites/40968/data/1670520678130-40968-0 last update time: Thu Dec 08 17:42:46 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 17:25:56 r2cloud java[18571]: Status: signal processing............................ [ru.r2cloud.satellite.decoder.APTDecoder$1 run]
Dec 11 17:26:01 r2cloud java[18571]: Gain: 16.0 [ru.r2cloud.satellite.decoder.APTDecoder$1 run]
Dec 11 17:26:01 r2cloud java[18571]: Channel A: 1 (visible) [ru.r2cloud.satellite.decoder.APTDecoder$1 run]
Dec 11 17:26:01 r2cloud java[18571]: Channel B: 4 (thermal infrared) [ru.r2cloud.satellite.decoder.APTDecoder$1 run]
Dec 11 17:26:05 r2cloud java[18571]: [1670778238517-28654-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:26:05 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/28654/data/1670778238517-28654-0/a.jpg [ru.r2cloud.satellite.ObservationDao saveImage]
Dec 11 17:26:05 r2cloud java[18571]: [1670778238517-28654-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:26:05 r2cloud java[18571]: [1670778238517-28654-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:26:05 r2cloud java[18571]: [1670775499438-40378-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:35:46 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 437269627 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/52950-1670798146924-52950-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 17:35:46 r2cloud java[18571]: [1670798146924-52950-0] moving rotator to Position [azimuth=215.79990391616053, elevation=4.998128360724716] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:35:46 r2cloud java[18571]: Found 1 device(s):
Dec 11 17:35:47 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 17:35:47 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 17:35:47 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 17:35:47 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 17:35:47 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 17:35:47 r2cloud java[18571]: Tuned to 437269627 Hz.
Dec 11 17:35:47 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 17:35:47 r2cloud java[18571]: Reading samples in async mode...
Dec 11 17:42:45 r2cloud java[18571]: [1670775499438-40378-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:42:45 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/40378/data/1670775499438-40378-0/data.bin [ru.r2cloud.satellite.ObservationDao saveData]
Dec 11 17:42:45 r2cloud java[18571]: [1670775499438-40378-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:42:46 r2cloud java[18571]: [1670775499438-40378-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 17:42:46 r2cloud java[18571]: [1670776168135-46839-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 17:44:56 r2cloud java[18571]: stopping: rtl_sdr for 1670798146924-52950-0 [ru.r2cloud.util.Util shutdown]
Dec 11 17:44:56 r2cloud java[18571]: Signal caught, exiting!
Dec 11 17:44:56 r2cloud java[18571]: [1670798146924-52950-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:44:57 r2cloud java[18571]: User cancel, exiting...
Dec 11 17:44:57 r2cloud java[18571]: [1670798146924-52950-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 17:44:57 r2cloud java[18571]: deleting old observation: ./data/satellites/46493/data/1670522939791-46493-0 last update time: Thu Dec 08 18:14:58 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 17:44:57 r2cloud java[18571]: deleting old observation: ./data/satellites/40042/data/1670523297424-40042-0 last update time: Thu Dec 08 18:24:28 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 17:56:36 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 435099874 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/51658-1670799396478-51658-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 17:56:36 r2cloud java[18571]: [1670799396478-51658-0] moving rotator to Position [azimuth=149.30555309933283, elevation=4.997069086052593] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 17:56:36 r2cloud java[18571]: Found 1 device(s):
Dec 11 17:56:36 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 17:56:36 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 17:56:36 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 17:56:37 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 17:56:37 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 17:56:37 r2cloud java[18571]: Tuned to 435099874 Hz.
Dec 11 17:56:37 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 17:56:37 r2cloud java[18571]: Reading samples in async mode...
Dec 11 18:02:25 r2cloud java[18571]: [1670776168135-46839-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:02:25 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/46839/data/1670776168135-46839-0/data.bin [ru.r2cloud.satellite.ObservationDao saveData]
Dec 11 18:02:25 r2cloud java[18571]: [1670776168135-46839-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:02:25 r2cloud java[18571]: [1670776168135-46839-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:02:25 r2cloud java[18571]: [1670774876746-43880-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:05:33 r2cloud java[18571]: stopping: rtl_sdr for 1670799396478-51658-0 [ru.r2cloud.util.Util shutdown]
Dec 11 18:05:33 r2cloud java[18571]: Signal caught, exiting!
Dec 11 18:05:34 r2cloud java[18571]: [1670799396478-51658-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:05:34 r2cloud java[18571]: User cancel, exiting...
Dec 11 18:05:34 r2cloud java[18571]: [1670799396478-51658-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 18:05:34 r2cloud java[18571]: deleting old observation: ./data/satellites/53951/data/1670524489403-53951-0 last update time: Thu Dec 08 18:44:08 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 18:14:32 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 437442973 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/41460-1670800472804-41460-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 18:14:32 r2cloud java[18571]: [1670800472804-41460-0] moving rotator to Position [azimuth=34.12496855546946, elevation=5.000067331728999] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:14:32 r2cloud java[18571]: Found 1 device(s):
Dec 11 18:14:32 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 18:14:32 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 18:14:33 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 18:14:33 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 18:14:33 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 18:14:33 r2cloud java[18571]: Tuned to 437442973 Hz.
Dec 11 18:14:33 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 18:14:33 r2cloud java[18571]: Reading samples in async mode...
Dec 11 18:15:57 r2cloud java[18571]: started with arguments: [rtl_test, -t] [ru.r2cloud.util.ProcessFactory create]
Dec 11 18:21:03 r2cloud java[18571]: [1670774876746-43880-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:21:03 r2cloud java[18571]: unable to save. dest already exist: /home/pi/r2cloud/./data/satellites/43880/data/1670774876746-43880-0/data.bin [ru.r2cloud.satellite.ObservationDao saveData]
Dec 11 18:21:03 r2cloud java[18571]: [1670774876746-43880-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:21:04 r2cloud java[18571]: [1670774876746-43880-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:21:04 r2cloud java[18571]: [1670778970469-43721-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:23:13 r2cloud java[18571]: loading satellites from leosatdata [ru.r2cloud.cloud.LeoSatDataClient loadSatellites]
Dec 11 18:23:14 r2cloud java[18571]: no satellite updates from leosatdata [ru.r2cloud.cloud.LeoSatDataClient loadSatellites]
Dec 11 18:23:14 r2cloud java[18571]: loading tle from: http://mstl.atl.calpoly.edu/~ops/keps/kepler.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:14 r2cloud java[18571]: received tle for 136 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:14 r2cloud java[18571]: loading tle from: http://celestrak.com/NORAD/elements/satnogs.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:15 r2cloud java[18571]: received tle for 252 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:15 r2cloud java[18571]: loading tle from: http://celestrak.com/NORAD/elements/active.txt [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:16 r2cloud java[18571]: received tle for 6847 satellites [ru.r2cloud.tle.CelestrakClient downloadTle]
Dec 11 18:23:16 r2cloud java[18571]: check for not processed observations [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670798146924-52950-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670784052694-40042-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670792393555-39444-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670799396478-51658-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670797065797-40014-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670784627898-46914-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670789782198-46912-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670780702937-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670794241800-43197-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670786458317-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670792247245-30776-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670795909835-46912-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670791605184-44419-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670779377193-43743-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670785351241-49017-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670793766986-43196-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670786919587-46493-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670794515469-52900-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670778970469-43721-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:16 r2cloud java[18571]: resuming decoding: 1670790683316-46914-0 [ru.r2cloud.satellite.decoder.DecoderService retryObservations]
Dec 11 18:23:19 r2cloud java[18571]: stopping: rtl_sdr for 1670800472804-41460-0 [ru.r2cloud.util.Util shutdown]
Dec 11 18:23:19 r2cloud java[18571]: [1670800771252-51657-0] moving rotator to Position [azimuth=173.88129292261826, elevation=51.681519718034004] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:23:19 r2cloud java[18571]: Signal caught, exiting!
Dec 11 18:23:19 r2cloud java[18571]: [1670800472804-41460-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:23:20 r2cloud java[18571]: User cancel, exiting...
Dec 11 18:23:20 r2cloud java[18571]: [1670800472804-41460-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 18:23:20 r2cloud java[18571]: deleting old observation: ./data/satellites/52770/data/1670525136889-52770-0 last update time: Thu Dec 08 18:55:01 UTC 2022 [ru.r2cloud.satellite.TimeSizeRetention ind
Dec 11 18:23:20 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 437520322 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/51657-1670800771252-51657-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 18:23:20 r2cloud java[18571]: Found 1 device(s):
Dec 11 18:23:20 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 18:23:20 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 18:23:20 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 18:23:20 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 18:23:20 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 18:23:20 r2cloud java[18571]: Tuned to 437520322 Hz.
Dec 11 18:23:21 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 18:23:21 r2cloud java[18571]: Reading samples in async mode...
Dec 11 18:28:54 r2cloud java[18571]: stopping: rtl_sdr for 1670800771252-51657-0 [ru.r2cloud.util.Util shutdown]
Dec 11 18:28:54 r2cloud java[18571]: Signal caught, exiting!
Dec 11 18:28:55 r2cloud java[18571]: [1670800771252-51657-0] observation time passed. cancelling rotation [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:28:56 r2cloud java[18571]: User cancel, exiting...
Dec 11 18:28:56 r2cloud java[18571]: [1670800771252-51657-0] rtl_sdr stopped: 143 [ru.r2cloud.satellite.reader.RtlSdrReader start]
Dec 11 18:35:17 r2cloud java[18571]: [1670778970469-43721-0] decoded [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:35:17 r2cloud java[18571]: [1670778970469-43721-0] uploading observation [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:35:19 r2cloud java[18571]: [1670778970469-43721-0] generating spectogram [ru.r2cloud.SpectogramService create]
Dec 11 18:38:32 r2cloud java[18571]: [1670778970469-43721-0] spectogram created [ru.r2cloud.SpectogramService create]
Dec 11 18:38:32 r2cloud java[18571]: [1670778970469-43721-0] observation uploaded [ru.r2cloud.cloud.LeoSatDataService uploadObservation]
Dec 11 18:38:32 r2cloud java[18571]: [1670779377193-43743-0] decoding [ru.r2cloud.satellite.decoder.DecoderService runInternally]
Dec 11 18:40:23 r2cloud java[18571]: started with arguments: ./etc/rtl_sdr_wrapper.sh -rtl rtl_sdr -f 401047880 -d 0 -s 240000 -g 40.0 -p 0 -o /tmp/46912-1670802023754-46912-0.raw.gz [ru.r2cloud.util.Proc
Dec 11 18:40:23 r2cloud java[18571]: [1670802023754-46912-0] moving rotator to Position [azimuth=302.9846386150321, elevation=5.000147865432726] [ru.r2cloud.satellite.RotatorService$1 run]
Dec 11 18:40:23 r2cloud java[18571]: Found 1 device(s):
Dec 11 18:40:23 r2cloud java[18571]:   0:  Realtek, RTL2838UHIDIR, SN: 00000001
Dec 11 18:40:23 r2cloud java[18571]: Using device 0: Generic RTL2832U OEM
Dec 11 18:40:24 r2cloud java[18571]: Found Rafael Micro R820T tuner
Dec 11 18:40:24 r2cloud java[18571]: [R82XX] PLL not locked!
Dec 11 18:40:24 r2cloud java[18571]: Sampling at 240000 S/s.
Dec 11 18:40:24 r2cloud java[18571]: Tuned to 401047880 Hz.
Dec 11 18:40:24 r2cloud java[18571]: Tuner gain set to 40.20 dB.
Dec 11 18:40:24 r2cloud java[18571]: Reading samples in async mode...

**Comment by dernasherbrezon** [comment]
It looks non-processed observations scheduled twice. Will work on a fix.

**Comment by dernasherbrezon** [comment]
Should be fixed in 5d92a9b78d0ab4b04eb8f8568e908cf52a0c2382
